andela/travel_tool_front
# Include request ID in the requests table

#### What does this PR do?
Includes the ID of requests in the requests table

#### Description of Task to be completed?
- add requestID header to the table headings
- modify the table rows to accept requests ID
- modify test to reflect the ID


#### How should this be manually tested?
- Clone the repo
- Enter the project's folder using `cd travel_tool_front`
- Run `./docker.sh development` to install dependencies and start the application
- Navigate to `localhost:3000/requests` in a browser to view the navigation pane and you should see the table with the request IDs

#### Any background context you want to provide?
None

#### What are the relevant pivotal tracker stories?
#159392486

#### Screenshots (if appropriate)
<img width="1440" alt="screen shot 2018-08-09 at 12 35 02 pm" src="https://user-images.githubusercontent.com/25195857/43896458-c771094e-9bd0-11e8-9770-476fee7978e0.png">

#### Questions:
N/A
